packages/base-styles/_variables.scss

```diff
@@ -37,7 +37,8 @@ $grid-unit-30: 3 * $grid-unit;		// 24px
 $grid-unit-40: 4 * $grid-unit;		// 32px
 $grid-unit-50: 5 * $grid-unit;		// 40px
 $grid-unit-60: 6 * $grid-unit;		// 48px
-
+$grid-unit-70: 7 * $grid-unit;    // 56px
```

Comment:
Nit: spaces are not consistent.

Reply:
Well spotted! I swapped spaces with tabs in https://github.com/WordPress/gutenberg/pull/39386/commits/1a89543ba69bd6d7461c3831e980efdee0aefc86